recipes-core/system-api/files/system-api-init

```diff
@@ -5,18 +5,19 @@
 # Required-Stop:     $network $remote_fs $syslog
 # Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
-# Short-Description: Start Orderflow Proxy at boot time
-# Description:       Enable Orderflow Proxy service.
+# Short-Description: Start System API at boot time
+# Description:       Enable System API service.
 ### END INIT INFO
 
 DAEMON=/usr/bin/system-api
 NAME=system-api
 DESC="System API Service"
-DAEMON_ARGS="--listen-addr 0.0.0.0:3535 --pipe-file /var/volatile/system-api.fifo"
+DAEMON_ARGS="--config /etc/system-api/systemapi-config.toml"
 PIDFILE=/var/run/$NAME.pid
 LOGFILE=/var/log/$NAME.log
 
 start() {
+    mkdir -p /persistent/rbuilder
```

Comment:
I see, this is assuming the new boot order, right?
If yes, I will backrun this PR with the new boot order we discussed last week

Reply:
makes sense to me, so uploads to that directory work even before rbuilder has started. not sure if that step shouldn't be part of the encrypted disk setup though, but seems fine.